src/sample_inverted_index.cpp

```diff
@@ -24,28 +26,71 @@ int main(int argc, char **argv)
     app.add_option("-o,--output", output_basename, "Output collection basename")->required();
     app.add_option("-r,--rate", rate, "Sampling rate (proportional size of the output index)")
         ->required();
+    app.add_option("-t,--type", type, "Sampling type")->required();
```

Comment:
Maybe the `type` option could show the supported samplings (`random_postings` and `random_docids`), explaining them briefly.